asu/routers/api.py

```diff
@@ -20,9 +20,21 @@ class BuildRequest(BaseModel):
     packages: Union[list, None] = []
     profile: str
     packages_versions: dict = {}
-    defaults: Union[str, None] = None
+    defaults: Optional[
+        Annotated[
+            str,
+            Field(default=None, max_length=20480, description="asu-defaults script"),
```

Comment:
It's called uci-defaults, just to avoid confusion :)

Reply:
I will go back to old openapi.yml and steal the comments from there.  Not sure why I didn't think of that sooner...

Reply:
Put the yaml comments into the new code.